src/main/java/no/unit/nva/cristin/projects/FetchCristinProjects.java

```diff
@@ -8,8 +8,6 @@
 import java.util.Map;
```

Comment:
In line 66 we have 

```
return createProjectsWrapper(language,title)
```
Rename method to describe what is actually being returned. The code should be readable like plain English almost.